roles/ceph-update/tasks/ceph_owner.yml

```diff
@@ -20,4 +20,4 @@
   when: "'ceph_osds_ssd' in group_names"
 
 - name: correct ceph dir owner
-  file: path=/var/lib/ceph state=directory owner=ceph group=ceph recurse=yes
+  script: files/ceph_owner.sh
```

Comment:
I have made more tests.
1. clean cache data(include memory cache, and cache in ssd tier)
2. run prep-ceph-chown.yml
3. run test playbook against ceph1 to chown with ansible `file` module, this takes 174.9 seconds
4. run test playbook against ceph2 to chown with our script, this takes 35.1 seconds